### PR TITLE
Update the Map benchmark to run cmd line in multi-user mode

### PR DESCRIPTION
**Pull request by peter-lawrey**
The 100 million entry test needed a little more memory than I had in graphical mode, by just enough in multi-user mode.

### CODE REVIEW

**Comment by peter-lawrey** [comment]
There are no changes to the core library, only a nested benchmark module

**Review by RobAustin — APPROVED** [review]

File: benchmark/src/main/java/net/openhft/chronicle/map/perf/MapJLBHTest$IFacadeSon$$Native.java [hunk 1]
```diff
@@ -0,0 +1,853 @@
+package net.openhft.chronicle.map.perf;
+
+import net.openhft.chronicle.bytes.*;
+import net.openhft.chronicle.values.Copyable;
+
+public class MapJLBHTest$IFacadeSon$$Native implements MapJLBHTest.IFacadeSon, Copyable<MapJLBHTest.IFacadeSon>, BytesMarshallable, Byteable {
```
Comment (JerryShea) [inline benchmark/src/main/java/net/openhft/chronicle/map/perf/MapJLBHTest$IFacadeSon$$Native.java:6]: are these generated? Do we need them?

**Review by JerryShea — APPROVED** [review]

File: benchmark/pom.xml [hunk 2]
```diff
@@ -89,6 +89,26 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+
+            <plugin>
```
Comment (JerryShea) [inline benchmark/pom.xml:93]: consider using `run-benchmarks` profile which is what I believe we use in other projects @alamar @nicktindall ?

**Review by JerryShea — APPROVED** [review]

File: benchmark/pom.xml (hunk 2 above)
Comment (nicktindall) [reply benchmark/pom.xml:93]: We do, I think there's a "standard" now in the scripted build config, it runs

`mvn clean install -Dskip.binary -DskipTests`
`mvn test -Prun-benchmarks -Dskip.binary -DuseIncrementalCompilation=false -DskipTests`

**Pull request merged**